### DISCUSSION TITLE
Docs for v 2.3.4

### DISCUSSION THREAD

**Discussion by nica0012** [react-native-image-picker/react-native-image-picker#1811] [General]
Hi there, 

I need to revert back to v2.3.4 because I need cropping abilities.

I went to look for the API but it seems like the page does not exist? 

Can somebody please point me there if you're aware of this location? on NPM I can see many people are reverting back to this version because of the no crop capability in the new versions.

Cheers!

**Comment by Johan-dutoit** [comment]
https://github.com/react-native-image-picker/react-native-image-picker/tree/2.3.4
